<ui version="4.0" >
 <class>QTwikeSimulatorClass</class>
 <widget class="QWidget" name="QTwikeSimulatorClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>QTwikeSimulator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item row="0" column="0" colspan="3" >
    <layout class="QGridLayout" name="gridLayout_2" >
     <item row="0" column="0" >
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QLCDNumber" name="lcdVoltage" />
       </item>
       <item>
        <widget class="QLabel" name="labelVolt" >
         <property name="text" >
          <string>Volt</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1" >
      <layout class="QHBoxLayout" name="horizontalLayout_5" >
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Sym:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdSym" />
       </item>
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Volt</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" >
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <item>
        <widget class="QLCDNumber" name="lcdCurrent" />
       </item>
       <item>
        <widget class="QLabel" name="labelAmpere" >
         <property name="text" >
          <string>Ampere</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0" >
      <layout class="QHBoxLayout" name="horizontalLayout_3" >
       <item>
        <widget class="QLCDNumber" name="lcdTemperature" />
       </item>
       <item>
        <widget class="QLabel" name="labelCelsius" >
         <property name="text" >
          <string>Â°C</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0" >
      <layout class="QHBoxLayout" name="horizontalLayout_4" >
       <item>
        <widget class="QLCDNumber" name="lcdCharge" />
       </item>
       <item>
        <widget class="QLabel" name="labelAh" >
         <property name="text" >
          <string>Ah</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="3" >
    <spacer name="horizontalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>198</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" >
    <spacer name="verticalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="4" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>dState</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QRadioButton" name="radio_drive" >
        <property name="text" >
         <string>Fahren</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QRadioButton" name="radio_off" >
        <property name="text" >
         <string>Off</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QRadioButton" name="radio_charge" >
        <property name="text" >
         <string>Laden</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QPushButton" name="button_start" >
     <property name="text" >
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>185</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2" colspan="2" >
    <widget class="QPushButton" name="button_stop" >
     <property name="text" >
      <string>Stop</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
